### PR TITLE
Add RockX to node providers

### PR DESCRIPTION
**Pull request by agol586**
**What changed? Why?**
    Add RockX to node providers

**Notes to reviewers**
  N/A

**How has it been tested?**
  Manually

### CODE REVIEW

File: apps/base-docs/docs/tools/node-providers.md [hunk 1]
```diff
@@ -192,6 +192,15 @@ hide_table_of_contents: true
 
 ---
 
+## RockX
+[RockX](https://access.rockx.com) offers a global blockchain node network and developer tools for Web3 innovation. Start with our free [Base RPC] (https://access.rockx.com/product/base-blockchain-api-for-web3-builders) to access institutional-grade solutions.
```
Comment (wbnns) [inline apps/base-docs/docs/tools/node-providers.md:196]: @agol586 

There's a missing space that's breaking the hyperlink for `Base RPC` could you please fix?

**Review by wbnns — CHANGES_REQUESTED** [review]
Small nit WRT link.

**Review by wbnns — APPROVED** [review]

File: apps/base-docs/docs/tools/node-providers.md [hunk 2]
```diff
@@ -192,6 +192,15 @@ hide_table_of_contents: true
 
 ---
 
+## RockX
+[RockX](https://access.rockx.com) offers a global blockchain node network and developer tools for Web3 innovation. Start with our free [Base RPC](https://access.rockx.com/product/base-blockchain-api-for-web3-builders) to access institutional-grade solutions.
```
Comment (briandoyle81CB) [inline apps/base-docs/docs/tools/node-providers.md:196]: Can we please say "onchain" instead of "Web3"?

**Review by briandoyle81CB — CHANGES_REQUESTED** [review]
Can we please say "onchain" instead of "Web3"?

**Comment by cb-heimdall** [comment]
Review Error for briandoyle81CB @ 2024-06-06 13:11:42 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by zencephalon — APPROVED** [review]

**Review by briandoyle81CB — APPROVED** [review]

**Pull request merged**